src/main/java/io/jenkins/docker/connector/DockerComputerJNLPConnector.java

```diff
@@ -195,10 +195,6 @@ public void beforeContainerCreated(DockerAPI api, String workdir, CreateContaine
         final String[] resolvedArgs = splitAndFilterEmpty(resolvedArgString, "\n");
 
         cmd.withCmd(resolvedArgs);
-        String vmargs = jnlpLauncher.vmargs;
-        if (StringUtils.isNotBlank(vmargs)) {
-            DockerEnvUtils.addEnvToCmd("JAVA_OPT", vmargs.trim(), cmd);
-        }
```

Comment:
As of jenkinsci/jenkins#6543 `vmargs` is no longer supported, so this is dead code (along with the corresponding tests).